ast/desugar/Desugar.cc

```diff
@@ -96,6 +96,18 @@ pair<MethodDef::ARGS_store, InsSeq::STATS_store> desugarArgs(DesugarContext dctx
                 destructures.emplace_back(node2TreeImpl(dctx, std::move(destructure)));
             } else if (auto *lhs = parser::cast_node<parser::Kwnilarg>(arg.get())) {
                 // TODO implement logic for `**nil` args
+            } else if (auto *fargs = parser::cast_node<parser::ForwardArg>(arg.get())) {
+                // we desugar (...) into (*<fwd-args>, **<fwd-kwargs>, &<fwd-block>)
+                // add `*<fwd-args>`
```

Comment:
Let's make it clearer that it's not just `(...)` (i.e. forward all parameters), but that this handles `(m, ...)` or `(m, n, ...)` too.  Maybe "We desugar `...` into `*<fwd-args>, **<fwd-kwargs>, &<fwd-block>`"?